charts/kamaji-crds/templates/kamaji.clastix.io_kubeconfiggenerators.yaml

```diff
@@ -1,10 +1,237 @@
+---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    cert-manager.io/inject-ca-from: {{ include "kamaji-crds.certManagerAnnotation" . }}
```

Comment:
This templating must be kept since it provides idempotence.

Reply:
Do you know if  this is a tool version issue as well? This wasn't intended.